lib/models/device.coffee

```diff
@@ -541,6 +659,21 @@ exports.restart = (uuid, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Restart application on device. *Deprecated*: Use
```

Comment:
Done. Also `@deprecated` should support a notice text after the tag (http://usejsdoc.org/tags-deprecated.html), yet it didn't seem to appear in the generated `DOCUMENTATION.md`